packages/cli/cli/package.json

```diff
@@ -114,5 +114,8 @@
     "vitest": "^2.1.4",
     "yaml": "^2.4.5",
     "yargs": "^17.4.1"
+  },
+  "dependencies": {
+    "@inquirer/prompts": "^7.1.0"
```

Comment:
lets put all dependencies in `devDependencies` (this is for `tsup` which we use as a bundling solution in the CLI)